centreon/src/Core/ServiceTemplate/Domain/Model/ServiceTemplate.php

```diff
@@ -139,21 +138,32 @@ public function __construct(
                 'name' => self::MAX_NAME_LENGTH,
                 'alias' => self::MAX_ALIAS_LENGTH,
                 'comment' => self::MAX_COMMENT_LENGTH,
-                'description' => self::MAX_DESCRIPTION_LENGTH,
                 'note' => self::MAX_NOTES_LENGTH,
                 'noteUrl' => self::MAX_NOTES_URL_LENGTH,
                 'actionUrl' => self::MAX_ACTION_URL_LENGTH,
                 'iconAlternativeText' => self::MAX_ICON_ALT_LENGTH,
             ] as $field => $limitation
         ) {
-            $propertyValue = ${$field};
+            $propertyValue = $this->{$field};
+            if ($field === 'alias') {
+                $propertyValue = preg_replace('/\s{2,}/', ' ', $propertyValue);
+                if ($propertyValue === null) {
+                    throw AssertionException::notNull($className . '::alias');
```

Comment:
Relates to comments on NewServiceTemplate
```suggestion
            if ($field === 'name') {
                $propertyValue = preg_replace('/\s{2,}/', ' ', $propertyValue);
                if ($propertyValue === null) {
                    throw AssertionException::notNull($className . '::name');
```